src/main/java/uk/gov/companieshouse/officer/delta/processor/processor/DeltaProcessor.java

```diff
@@ -61,7 +62,8 @@ public void process(ChsDelta delta) throws RetryableErrorException, NonRetryable
                 OfficersItem officer = officersList.get(i);
                 DataMapHolder.get()
                         .companyNumber(officer.getCompanyNumber())
-                        .officerId(officer.getOfficerId());
+                        .officerId(officer.getOfficerId())
+                        .internalId(officer.getInternalId());
 
                 logMap = DataMapHolder.getLogMap();
                 logger.infoContext(logContext, String.format("Process data for officer [%d]", i), logMap);
```

Comment:
This log message is redundant I think, it just logs the index of the current officer which will always be 0.

Reply:
Gone. I also refactored the redundant `logMap` parameter passing